.github/workflows/odoo-ci-cd.yml

```
@@ -2,40 +1,267 @@
+          # Remove trailing comma
+          DOCKER_INSTANCES="${DOCKER_INSTANCES%,}"
+
+          echo "Final DOCKER_INSTANCES: $DOCKER_INSTANCES"
+
+          # Export for use in later steps
+          echo "docker_instances=$DOCKER_INSTANCES" >> "$GITHUB_OUTPUT"
+
+      - name: Debug Odoo Module Installation
+        run: |
+          echo "🔍 Checking Odoo module installation..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)